db/structure.sql

```diff
@@ -160,10 +160,10 @@ SET default_table_access_method = heap;
 --
 
 CREATE TABLE public.ar_internal_metadata (
```

Comment:
All these extra spaces should be removed. You should be able to get the original file from main by running
```
git checkout origin/main db/structure.sql
```